### DISCUSSION TITLE
F-Droid releases of RiMusic

### DISCUSSION THREAD

**Discussion by sergeimikhailov** [fast4x/RiMusic#4355] [Q&A]
Why is F-Droid's version is almost 3 weeks behind?
Is there a publishing pipeline broken somewhere?

**Comment by fast4x** [comment]
Release flow is different, FDroid build from scratch normally after 1/2 weeks.

**Comment by ChiefMikeK** [comment]
[IzzyOnDroid F-Droid Repository](https://apt.izzysoft.de/fdroid/index.php) is a faster update

**Reply by fast4x** [reply]
Yes, RiMusic is inside Izzy, but someone want specific fdroid build, in this case must wait for this

**Comment by twistios** [comment]
F-Droid was updated yesterday with version 0.6.59.
